modules/registry-configuring-storage-baremetal.adoc

```diff
@@ -3,6 +3,7 @@
 // * installing/installing_bare_metal/installing-bare-metal.adoc
 // * installing/installing_restricted_networks/installing-restricted-networks-bare-metal.adoc
 // * registry/configuring-registry-storage-baremetal
+// * installing/installing_ibm_z/installing-ibm-z.adoc
```

Comment:
Unrelated to this change specifically, but on line 18 below, could we add a link to the PV configuration documentation:
https://docs.openshift.com/container-platform/4.2/storage/persistent-storage/persistent-storage-nfs.html

Reply:
Due to a frustrating tooling restriction, we can't add an internal link to modules.